src/main/scala/ru/otus/sc/countdown/model/CountdownRequest.scala

```diff
@@ -0,0 +1,6 @@
+package ru.otus.sc.countdown.model
+
+sealed trait CountdownRequest
+
+case class CountdownClearRequest(initValue: Long = 1) extends CountdownRequest
+case class CountdownTickRequest()                     extends CountdownRequest
```

Comment:
```suggestion
case object CountdownTickRequest                      extends CountdownRequest
```

Класс без параментов это синглтон.

Reply:
буду знать, спасибо